electron/preload/index.ts

```diff
@@ -48,7 +48,8 @@ type Commands = {
   ): Promise<BackupKeyRecoveryTransansaction | FormattedOfflineVaultTxInfo>;
   setBitGoEnvironment(
     environment: 'prod' | 'test',
-    apiKey?: string
+    apiKey?: string,
+    apiKeyCoin?: string
```

Comment:
use the same API signature as we do in every other `coin` specific API call.

coin, environment, apiKey